CMakeLists.txt

```diff
@@ -418,10 +420,14 @@ message(STATUS "Generated HIP_OFFLOAD_LINK_OPTIONS: ${HIP_OFFLOAD_LINK_OPTIONS_I
 # Must separate these targets into deviceInteral to be used for building samples,etc.
 # and device for external export. Have to do this because generator expressions don't seem to work here.
 # TODO: Is there a better way to do this?
+
+set(CHIP_SPV_OFFLOAD_COMPILE_OPTIONS  -x hip --target=x86_64-linux-gnu)
+
 add_library(deviceInternal INTERFACE)
 target_compile_options(deviceInternal INTERFACE
   ${HIP_ENABLE_SPIRV}
   ${HIP_OFFLOAD_COMPILE_ONLY_OPTIONS_}
+  ${CHIP_SPV_OFFLOAD_COMPILE_OPTIONS}
```

Comment:
Should have better naming? It seems the `CHIP_SPV_OFFLOAD_COMPILE_OPTIONS` is indistinguishable from `HIP_OFFLOAD_COMPILE_ONLY_OPTIONS_`.